api/apps/api/test/scenarios/start-scenario-calibration.e2e-spec.ts

```diff
@@ -30,6 +30,14 @@ describe('start-scenario-calibration', () => {
     await fixtures.ThenWhenReadingProjectCalibrationItHasTheNewRange();
   });
 
+  it(`throws an exception if an export is running`, async () => {
+    await fixtures.GivenScenarioWasCreated();
+
+    await fixtures
+      .ThenShouldFailWhenStartingAnScenarioCalibrationWithA()
```

Comment:
minor and from earlier changes, but I suggest to take this chance to fix the name of this fixture method

```suggestion
      .ThenShouldFailWhenStartingAScenarioCalibrationWithA()
```